lib/cli/generators/UPDATE_PACKAGE_ORGANIZATIONS/index.js

```diff
@@ -0,0 +1,50 @@
+const helpers = require('../../lib/helpers');
+const latestVersion = require('latest-version');
+const spawn = require('child-process-promise').spawn;
+const path = require('path');
+
+const packageNames = require('@storybook/codemod').packageNames;
+
+function updatePackage(devDependencies, oldName, newName) {
+  if (devDependencies[oldName]) {
+    return latestVersion(newName).then(version => {
+      delete devDependencies[oldName];
+      devDependencies[newName] = version;
+    });
+  } else {
+    return Promise.resolve(null);
+  }
+}
+
+function updatePackageJson() {
+  const packageJson = helpers.getPackageJson();
+  const { devDependencies } = packageJson;
+
+  return Promise.all(
+    Object.entries(packageNames).map(([oldName, newName]) =>
+      updatePackage(devDependencies, oldName, newName)
+    )
+  ).then(() => {
+    if (!devDependencies['@storybook/react'] && !devDependencies['@storybook/react-native']) {
+      throw new Error('Expected to find `@kadira/[react-native]-storybook` in devDependencies');
+    }
+    helpers.writePackageJson(packageJson);
+  });
+}
+
+function updateSourceCode() {
+  const jscodeshiftPath = path.dirname(require.resolve('jscodeshift'));
```

Comment:
No, but it feels a little hacky. For instance it hard codes the path of the binary into this file (rather than `npm run jscodeshift` which uses the `bin` field of `jscodeshift`'s `package.json`).